src/heartbeat.ts

```diff
@@ -92,12 +96,20 @@ export class HeartbeatManager extends Disposable {
 
             this.sendHeartBeat();
         }, HeartbeatManager.HEARTBEAT_INTERVAL);
+
+        this.eventCounterHandle = setInterval(() => {
+            this.telemetry.sendRawTelemetryEvent('vscode_desktop_user_session_events', { events: Object.fromEntries(this.eventCounterMap) });
```

Comment:
(nit) `vscode_desktop_heartbeat_sum` or something related to heartbeat aggregation